.github/workflows/create-demo-clusters.yml

```diff
@@ -428,6 +433,7 @@ jobs:
           repository: stackrox/${{ inputs.kube-burner-config-repo }}
           path: .kube-burner-config
           ref: ${{ inputs.kube-burner-config-ref }}
+      # TODO(ROX-29223): Remove once old versions don't use the benchmark-operator
```

Comment:
When will that be? Which version is the last to use the benchmark-operator?

Reply:
If it makes sense, we can make a new major version of our actions, ie. `v2` to use in newer releases.

Reply:
The last version to use the benchmark-operator is 4.7. Once 4.7 is out of support this can be removed. Also we can backport some changes to release branches and that way we can remove this faster.

Reply:
I think we should probably be tagging the actions with the next release. I am actually working of a design proposal for how to evolve these actions.